react-exercise/src/components/filter-buttons/FilterButtons.tsx

```diff
@@ -0,0 +1,30 @@
+import "./FilterButtons.scss";
+import { FilterType } from "../../types/filter.ts";
+import clsx from "clsx";
+
+interface FilterButtonsProps {
+  activeFilter?: FilterType;
+  onFilterClick: (filter: FilterType) => void;
+}
+
+const filters: FilterType[] = ["Cities", "Age"];
```

Comment:
Note that you can create this list from FilterType directly, and this way to insure that all values are in the array:
export const filters = ["Cities", "Age"] as const;
export type FilterType = (typeof filters)[number];